hack/make/binary

```diff
@@ -52,8 +52,14 @@ if [ "$IAMSTATIC" == "true" ] && [ "$(go env GOHOSTOS)" == "linux" ] && [ "$DOCK
 	fi
 fi
 
+REBUILD_FLAG="-a"
+if [ "$DOCKER_INCREMENTAL_BINARY" ]; then
```

Comment:
But, shouldn't this block be in `make.sh` and added to `ORIG_BUILDFLAGS` since that applies to tests too?